sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/src/Azure.ResourceManager.NeonPostgres.csproj

```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0-beta.2</Version>
+    <Version>1.0.0-beta.1</Version>
```

Comment:
the version 1.0.0-beta.1 has been released.
we should revert this back.

Reply:
Done